### ISSUE TITLE
Incorrect dependency in diet4j-examples-activate

### ISSUE THREAD

**Issue by argv-minus-one** [diet4j/diet4j-examples#1]
diet4j-examples-activate depends on diet4j-core `${project.version}`. This does not work because the current master version of diet4j is 0.8, and the current release version is 0.7 (not snapshot).

In general, dependencies on artifacts that are not part of the same project should not use `${project.version}` as the dependency version. Since diet4j-examples is not part of diet4j proper, that applies here. Instead, the dependency should be explicit (0.7 or 0.8-SNAPSHOT).

**Comment by jernst** [comment]
Fixed in a00e3e940f42972de5f99623a98b871459b43f16
